src/solver/optimisation/constraints/Group1.cpp

```diff
@@ -0,0 +1,86 @@
+#include "Group1.h"
+AreaBalanceData Group1::GetAreaBalanceData()
+{
+    return {.CorrespondanceCntNativesCntOptim = problemeHebdo_->CorrespondanceCntNativesCntOptim,
+            .IndexDebutIntercoOrigine = problemeHebdo_->IndexDebutIntercoOrigine,
+            .IndexSuivantIntercoOrigine = problemeHebdo_->IndexSuivantIntercoOrigine,
+            .IndexDebutIntercoExtremite = problemeHebdo_->IndexDebutIntercoExtremite,
+            .IndexSuivantIntercoExtremite = problemeHebdo_->IndexSuivantIntercoExtremite,
+            .PaliersThermiquesDuPays = problemeHebdo_->PaliersThermiquesDuPays,
+            .ShortTermStorage = problemeHebdo_->ShortTermStorage};
+}
+
+FictitiousLoadData Group1::GetFictitiousLoadData()
+{
+    return {.CorrespondanceCntNativesCntOptim = problemeHebdo_->CorrespondanceCntNativesCntOptim,
+            .PaliersThermiquesDuPays = problemeHebdo_->PaliersThermiquesDuPays,
+            .DefaillanceNegativeUtiliserHydro = problemeHebdo_->DefaillanceNegativeUtiliserHydro};
+}
+
+ShortTermStorageLevelData Group1::GetShortTermStorageLevelData()
+{
+    return {
+      .CorrespondanceCntNativesCntOptim = problemeHebdo_->CorrespondanceCntNativesCntOptim,
+      .ShortTermStorage = problemeHebdo_->ShortTermStorage,
+    };
+}
+
+FlowDissociationData Group1::GetFlowDissociationData()
+{
+    return {.CorrespondanceCntNativesCntOptim = problemeHebdo_->CorrespondanceCntNativesCntOptim,
+            .CoutDeTransport = problemeHebdo_->CoutDeTransport,
+            .PaysOrigineDeLInterconnexion = problemeHebdo_->PaysOrigineDeLInterconnexion,
+            .PaysExtremiteDeLInterconnexion = problemeHebdo_->PaysExtremiteDeLInterconnexion};
+}
+
+BindingConstraintHourData Group1::GetBindingConstraintHourData()
+{
+    return
+
+      {.MatriceDesContraintesCouplantes = problemeHebdo_->MatriceDesContraintesCouplantes,
+       .CorrespondanceCntNativesCntOptim = problemeHebdo_->CorrespondanceCntNativesCntOptim,
+       .PaliersThermiquesDuPays = problemeHebdo_->PaliersThermiquesDuPays};
+}
+
+void Group1::Build()
+{
+    auto areaBalanceData = GetAreaBalanceData();
+    AreaBalance areaBalance(builder_, areaBalanceData);
+
+    auto fictitiousLoadData = GetFictitiousLoadData();
+    FictitiousLoad fictitiousLoad(builder_, fictitiousLoadData);
+
+    auto shortTermStorageLevelData = GetShortTermStorageLevelData();
+    ShortTermStorageLevel shortTermStorageLevel(builder_, shortTermStorageLevelData);
+
+    auto flowDissociationData = GetFlowDissociationData();
+    FlowDissociation flowDissociation(builder_, flowDissociationData);
+
+    auto bindingConstraintHourData = GetBindingConstraintHourData();
+    BindingConstraintHour bindingConstraintHour(builder_, bindingConstraintHourData);
```

Comment:
```c++
auto areaBalanceData = GetAreaBalanceData();
AreaBalance areaBalance(builder_, areaBalanceData);
...
```
As said in **Group1.h** 's review, this block (constraints construction) should be removed.

Reply:
group of constraints are not returned in anyway for any reason, i don't find why we would do that